src/cargo.rs

```diff
@@ -250,14 +250,21 @@ impl Root {
     }
 }
 
-pub fn root(mode: XargoMode) -> Result<Option<Root>> {
+pub fn root(mode: XargoMode, path: Option<&str>) -> Result<Option<Root>> {
```

Comment:
Maybe call this  `root` or `root_dir` or so, for consistency with `Args`.

Reply:
fixed